HiddenThread.user.js

```diff
@@ -1304,72 +1068,114 @@ function loadHiddenPosts()
+                <p style="font-size:x-large;">Скрытотред v0.1</p>
+                <span id="hiddenThreadToggle" style="position: absolute; right: 0; cursor: pointer">${toggleText()}</span>
             </div>
-            <textarea
-                id="hiddenPostInput"
-                placeholder="Пиши скрытый текст тут"
-                style="box-sizing: border-box; display: inline-block; width: 100%; padding: 5px;"
-                rows="10"
-            ></textarea>
-            <div id="hiddenFilesDiv" style="padding: 5px;">
-                <span>Выбери скрытые файлы: </span>
-                <input id="hiddenFilesInput" type="file" multiple="true" />
-                <br>
-                <span>Выбери картинку-контейнер: </span>
-                <input id="hiddenContainerInput" type="file">
-                <br>
-                <input id="hiddenFilesClearButton" type="button" value="Очистить список файлов">
-            </div>
-            <div style="padding: 5px;">
-                <div style="font-size:large;text-align:center;">Подписать пост</div>
-                Приватный ключ (ECDSA p256, base58): <br>
-                <input
-                    id="privateKey"
-                    style="box-sizing: border-box; display: inline-block; width: 100%; padding: 5px;"
-                />
-                <br>
-                Публичный ключ:
-                <br>
-                <input
-                    id="publicKey"
-                    readonly
+            <div id="hiddenThreadForm" style="display: ${storage.hide ? 'none' : ''}">
+                <div style="padding:5px;">
+                    <span style="padding-right: 5px;">Пароль:</span>
+                    <input id="hiddenThreadPassword" /> 
+                    <input id="loadHiddenPostsButton" type="button" style="padding: 5px;" value="Загрузить скрытопосты" />
+                    <!--<input id="clearLoadedPosts" type="button" style="padding: 5px;" value="X" />-->
+                </div>
+                <div style="padding:5px;text-align:center;">
+                    <!--<span id="loadingStatus" style="display: none">Загрузка...</span>-->
+                    Загружено картинок: <span id="imagesLoadedCount">0</span>/<span id="imagesCount">0</span>
+                    <br>
+                    Загружено скрытопостов: <span id="hiddenPostsLoadedCount">0</span>
+                </div>
+                <textarea
+                    id="hiddenPostInput"
+                    placeholder="Пиши скрытый текст тут"
                     style="box-sizing: border-box; display: inline-block; width: 100%; padding: 5px;"
-                />
+                    rows="10"
+                ></textarea>
+                <div id="hiddenFilesDiv" style="padding: 5px;">
+                    <span>Выбери скрытые файлы: </span>
+                    <input id="hiddenFilesInput" type="file" multiple="true" />
+                    <span>Выбери картинку-контейнер: </span>

```

Comment:
Надо оставить перенос, иначе часть текста остается на первой строке
```suggestion
                    <br>
                    <span>Выбери картинку-контейнер: </span>

```